specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2020-04-01-preview/EventGrid.json

```diff
@@ -5896,9 +5896,32 @@
             "modelAsString": true
           }
         },
+        "partnerTopicReadinessState": {
+          "description": "The readiness state of the corresponding partner topic.",
+          "enum": [
+            "NotActivatedByUserYet",
+            "ActivatedByUser",
+            "DeactivatedByUser",
+            "DeletedByUser"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PartnerTopicReadinessState",
+            "modelAsString": true
+          }
+        },
+        "expirationTimeIfNotActivatedUtc": {
```

Comment:
>expirationTimeIfNotActivatedUtc [](start = 9, length = 31)

Can this parameter name simplified as expirationTime? Add UTC in the description explaining what does it mean?

Reply:
Suggest reviewing all the parameters with the PM as this would be public facing contract.

---
In reply to: [423423487](https://github.com/Azure/azure-rest-api-specs/pull/9402#discussion_r423423487) [](ancestors = 423423487)

Reply:
This has been reviewed internally. Thanks. and we would like to use it as is because we have different concept of expiration time in event subscription that is different from this partner topic activation expiration time. We need to differential and be specific. thanks